dev/Repeater/APITests/FlowLayoutTests.cs

```diff
@@ -1601,22 +1601,29 @@ private void ValidateLayoutEstimations(ScrollOrientation scrollOrientation, Layo
                         break;
 
                     case LayoutChoice.Grid:
-                        var minRowSpacing = om.ScrollOrientation == ScrollOrientation.Vertical ? lineSpacing : 0;
-                        var minColumnSpacing = om.ScrollOrientation == ScrollOrientation.Horizontal ? lineSpacing : 0;
-                        layout = new UniformGridLayout() { MinItemWidth = itemSize, MinItemHeight = itemSize, MinRowSpacing = minRowSpacing, MinColumnSpacing = minColumnSpacing };
+                        {
+                            var minRowSpacing = om.ScrollOrientation == ScrollOrientation.Vertical ? lineSpacing : 0;
+                            var minColumnSpacing = om.ScrollOrientation == ScrollOrientation.Horizontal ? lineSpacing : 0;
+                            layout = new UniformGridLayout() { MinItemWidth = itemSize, MinItemHeight = itemSize, MinRowSpacing = minRowSpacing, MinColumnSpacing = minColumnSpacing };
+                        }
                         break;
 
                     case LayoutChoice.Flow:
-                        layout = new FlowLayoutDerived()
                         {
-                            MinColumnSpacing = lineSpacing,
```

Comment:
There seem to be a lot of unrelated changes in the PR. can you merge with master and force update your branch/PR ?